doc/releases/changelog-0.39.0.md

```diff
@@ -477,6 +477,9 @@
 * Fixes a bug where `default.tensor` raises an error when applying `Identity`/`GlobalPhase` on no wires, and `PauliRot`/`MultiRZ` on a single wire.
   [(#6448)](https://github.com/PennyLaneAI/pennylane/pull/6448)
 
+* Fixes a bug where `qml.ctrl` and `qml.adjoint` queued extra operators if they were defined as the arguments.
```

Comment:
```suggestion
* Fixes a bug where applying `qml.ctrl` and `qml.adjoint` on an operator type instead of an operator instance results in extra operators in the queue.
```